content/appleboy/drone-line/index.md

```diff
@@ -132,22 +131,24 @@ pipeline:
     channel_secret: xxxxxxxxxx
     channel_token: xxxxxxxxxx
     to: line_user_id
-+   message: |
-+     {{ #success build.status }}
-+       build {{ build.number }} succeeded. Good job.
-+     {{ else }}
-+       build {{ build.number }} failed. Fix me please.
-+     {{ /success }}
++   message: >
++     {{#success build.status}}
++       build {{build.number}} succeeded. Good job.
++     {{else}}
++       build {{build.number}} failed. Fix me please.
++     {{/success}}
 ```
 
-# Parameter Reference
+# Secret Reference
 
 channel_secret
```

Comment:
should this also include `line_channel_secret`?

Reply:
Done!

Reply:
see https://github.com/drone/drone-plugin-index/pull/112/commits/0ee8253f3d2f8fdcfb2944bd6b53720f667cf6bf